dev/tests/acceptance/tests/functional/Magento/FunctionalTest/Config/Page/AdminConfigPage.xml

```diff
@@ -12,4 +12,7 @@
     <page name="AdminContentManagementPage" url="admin/system_config/edit/section/cms/" area="admin" module="Magento_Config">
         <section name="ContentManagementSection"/>
     </page>
+    <page name="AdminCatalogInventoryConfigPage" url="admin/system_config/edit/section/cataloginventory" area="admin" module="Magento_Config">
```

Comment:
This also should be moved to `Magento_CatalogInventory`

Reply:
Done.